tsconfig.json

```diff
@@ -15,10 +15,6 @@
     "sourceMap": false,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
-    "typeRoots": [
-      "node_modules/@types",
```

Comment:
Do you check if there is no error with ESLint in your IDE when a file of the tests is updated ?
In my memories, we added this for the edition of the tests on IDE.

Due to the bug on IntelliJ on macOS, ESLint doesn't work on the edition. So, I can't verify 🙁

Reply:
✔️ Yes it works. If I add an unsued import
```typescript
// in src/component/BpmnVisualization.ts, RenderedModel is unused
import { BpmnModelRegistry, RenderedModel } from './registry/bpmn-model-registry';
```

I get a `typescript-eslint` error:
```
/development/process-analytics/repo/bpmn-visualization-js/src/component/BpmnVisualization.ts
  24:29  warning  'RenderedModel' is defined but never used  @typescript-eslint/no-unused-vars
```

The default `typeRoots` value is `["node_modules/@types"]`. We overrode it to add the mxGraph types with `"node_modules/@typed-mxgraph"`